### PR TITLE
feat: add less loader and add example styles

### PR DESCRIPTION
**Pull request by tripodsan**
## What I did
- adding less loader
- adding example styles from https://github.com/adobe/aem-core-wcm-components/tree/master/examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/styles
- replacing progress bar knob with slider:

image

image

### CODE REVIEW

File: app/aem/src/server/framework-preset-aem.ts [hunk 1]
```diff
@@ -41,6 +41,11 @@ export function webpack(config: Configuration) {
             },
           ],
         },
+        {
```
Comment (CodeByAlex) [inline app/aem/src/server/framework-preset-aem.ts:44]: awesome!

**Comment by CodeByAlex** [comment]
I really like the addition of the less loader as it will fix problems for users who want to develop based on the aem standard. I get having a demo js file or less file but why are we adding in so many js and style sheet files?

**Comment by tripodsan** [comment]
> I really like the addition of the less loader as it will fix problems for users who want to develop based on the aem standard. I get having a demo js file or less file but why are we adding in so many js and style sheet files?

I can create 2 PRs if you like.

the styles are from the [aem-core-components](https://github.com/adobe/aem-core-wcm-components/tree/master/examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/styles) repository and are examples. I thought it might be nicer to look at the components styled.

if there is a way to select 'themes' in storybook, that would be a better way to do it.

**Comment by CodeByAlex** [comment]
It definitely makes sense to show them with various styles - i would think this would work - what do you think? https://github.com/CodeByAlex/storybook-theme-knob

**Comment by tripodsan** [comment]
> It definitely makes sense to show them with various styles - i would think this would work - what do you think? https://github.com/CodeByAlex/storybook-theme-knob
> It allows one to many themes and can be modified by story since it’s a knob

if you could add this, that would be great

**Comment by CodeByAlex** [comment]
Yeah I can do that - I’ll add it in a subsequent pr

**Pull request merged**